CMakeLists.txt

```diff
@@ -143,6 +143,9 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/ll
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
+set_var_to_condition(MIOPEN_OFFLINE_COMPILER_PATHS_V2_DEFAULT FALSE)
```

Comment:
This is overengineering. Instead, use `OFF` in the `option()` command below.

Reply:
@apwojcik FYI in the future the value of the _DEFAULT will be computed based on platform (Win32 or Linux) and ROCm version. However I am not against your comment; we can always add this stuff later.